src/components/package-custom-coverage/package-custom-coverage.js

```diff
@@ -133,78 +61,101 @@ class PackageCustomCoverage extends Component {
     this.props.initialize(this.props.initialValues);
   }
 
-  render() {
-    let { pristine, isPending, handleSubmit, onSubmit } = this.props;
-    let { intl } = this.context;
-    let contents;
-    let { customCoverages } = this.props.initialValues;
+  renderEditingForm() {
+    let {
+      pristine,
+      isPending,
+      handleSubmit,
+      onSubmit
+    } = this.props;
 
-    if (this.state.isEditing) {
-      contents = (
-        <form
-          data-test-eholdings-custom-coverage-form
-          onSubmit={handleSubmit(onSubmit)}
-        >
-          <FieldArray name="customCoverages" component={this.renderCoverageFields} />
-          <div className={styles['custom-coverage-action-buttons']}>
-            <div
-              data-test-eholdings-package-details-cancel-custom-coverage-button
-              className={styles['custom-coverage-action-button']}
+    return (
+      <form
+        data-test-eholdings-custom-coverage-form
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <PackageCoverageFields />
+        <div className={styles['custom-coverage-action-buttons']}>
+          <div
+            data-test-eholdings-package-details-cancel-custom-coverage-button
+            className={styles['custom-coverage-action-button']}
+          >
+            <Button
+              disabled={isPending}
+              type="button"
+              role="button"
```

Comment:
Is `role="button"` necessary here? (it's already a `<button>`)

I know this is what was previously here, but while we're at it we can clean it up a little.

Unless there is a reason this button element needs to explicitly have a button role?

Reply:
Probably not needed. Removed everywhere we were doing that with a `Button`.